packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx

```diff
@@ -189,8 +178,12 @@ const BpkInfoBannerInner = ({
       {...rest}
     >
       <section className={sectionClassNames} role="presentation">
-        <div
+        <BannerHeader
           role={isExpandable ? 'button' : undefined}
```

Comment:
role button is not needed on a <button>, you might want to set type="button" when expandable :D